canonical-web-and-design/snapcraft.io
# Unreleased tracks

## Done

- Added tracks as option to release state

## Issue / Card

Fixes #2869 

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit http://0.0.0.0:8004/bitcoin-core/releases - you should be able to select v 0.2.0 from the tracks dropdown and it shouldn't contain any releases.
